assets/react/v3/shared/components/modals/Modal.tsx

```diff
@@ -1,4 +1,4 @@
-import { colorTokens, zIndex } from '@Config/styles';
+import { colorTokens, zIndex as configZIndex } from '@Config/styles';
```

Comment:
Why? Could you please explain?

Reply:
As there was two same variables, but this would not be making any issue now as I have changed the other one to be `depthIndex` and kept it as `zIndex`.